### ISSUE TITLE
yarn is broken in production

### ISSUE THREAD

**Issue by cebe** [yiisoft-contrib/yiiframework.com#434]
```
----------
          ID: yarn_install
    Function: cmd.run
        Name: yarn install
      Result: False
     Comment: Command "yarn install" run
     Started: 09:36:56.937360
    Duration: 450.598 ms
     Changes:   
              ----------
              pid:
                  16411
              retcode:
                  1
              stderr:
                  /usr/share/yarn/lib/cli.js:45726
                    let {
                        ^
                  
                  SyntaxError: Unexpected token {
                      at exports.runInThisContext (vm.js:53:16)
                      at Module._compile (module.js:373:25)
                      at Object.Module._extensions..js (module.js:416:10)
                      at Module.load (module.js:343:32)
                      at Function.Module._load (module.js:300:12)
                      at Module.require (module.js:353:17)
                      at require (internal/module.js:12:17)
                      at Object.<anonymous> (/usr/share/yarn/bin/yarn.js:24:13)
                      at Module._compile (module.js:409:26)
                      at Object.Module._extensions..js (module.js:416:10)
              stdout:
```

**Comment by machour** [comment]
probably a node version issue, see https://github.com/yarnpkg/yarn/issues/6914

**Comment by cebe** [comment]
quite likely but they broke BC on the stable branch -.-

Need to check if we can force some versions.

**Comment by cebe** [comment]
fixed by setting yarn version explictly to `1.12.3-1`.
